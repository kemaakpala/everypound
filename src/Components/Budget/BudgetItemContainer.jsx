import Grid from '@material-ui/core/Grid';
import Paper from '@material-ui/core/Paper';
import Button from '@material-ui/core/Button';
import Divider from '@material-ui/core/Divider';
import { makeStyles } from '@material-ui/core/styles';
import BudgetItemActions from './BudgetItemActions';
const useStyles = makeStyles((theme) => ({
  root: {
    marginBottom: theme.spacing(1),
    padding: theme.spacing(1)
  }
}))
const BudgetItemContainer = ({ children }) => {
  const classes = useStyles();
  return (
    <Paper className={classes.root} elevation={0} variant="outlined">
      <Grid container spacing={2}
      // style={{ border: '1px solid black', margin: '8px 0 0 0' }}
      >
        <Grid item xs={12}
        // style={{ border: '4px solid green' }}
        > {children}</Grid>
        <BudgetItemActions />
      </Grid>
    </Paper>
  )
}

export default BudgetItemContainer;