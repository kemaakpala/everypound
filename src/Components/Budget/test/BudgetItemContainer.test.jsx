import BudgetItemContainer from '../BudgetItemContainer';
import { render } from '@testing-library/react';

describe('<BudgetItemContainer />', () => {
  it('matches snapshot', () => {
    const { firstChild } = render(<div><BudgetItemContainer /></div>)
    expect(firstChild).toMatchSnapshot();
  })
})