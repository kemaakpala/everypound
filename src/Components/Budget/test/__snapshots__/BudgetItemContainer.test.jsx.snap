// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BudgetItemContainer /> matches snapshot 1`] = `undefined`;
